sample-xml/app/src/test/java/co/nimblehq/sample/xml/ui/screens/xml/HomeFragmentTest.kt

```diff
@@ -23,16 +31,71 @@ class HomeFragmentTest : BaseFragmentTest<HomeFragment, FragmentHomeBinding>() {
     var hiltRule = HiltAndroidRule(this)
 
     @Before
-    fun setup() {
+    fun setUp() {
         hiltRule.inject()
     }
 
     @Test
-    fun `When initializing fragment, it displays the recycler view`() {
+    fun `When launching fragment, it displays the recycler view`() {
```

Comment:
Should we also bring these updates to `template-xml` project?

Reply:
Added in this https://github.com/nimblehq/android-templates/pull/450/commits/da634200a4bfe703fc2e947435c7f2ebf0c76c8d ✅